packages/scoutgame/src/builderNfts/recordNftMint.ts

```diff
@@ -288,6 +289,13 @@ export async function recordNftMint(
     log.error('Error recording referral bonus', { error, builderId: builderNft.builderId, scoutId });
   }
 
+  refreshEstimatedPayouts({
+    week: getCurrentWeek(),
+    builderIdToRefresh: builderNft.builderId
+  }).catch((error) => {
+    log.error('Error refreshing estimated payouts', { error, builderId: builderNft.builderId });
```

Comment:
use userId, not builderId for logs

Reply:
Agreed & fixed. BuilderId as standard, and userId for the scout actually minting